tox.ini

```diff
@@ -16,6 +16,11 @@ max_line_length = 100
 [pytest]
 addopts = --cov=message_tagging_service
 
+[travis]
+python =
+  3.6: py36, flake8
+  3.7: py37
```

Comment:
", flake8"?

Reply:
Yeah. I updated `.travis-ci.yml`. It runs flake8 on every supported Python version.